### ISSUE TITLE
[Ruchy] Cannot find by waypoint

### ISSUE THREAD

**Issue by kumy** [geokrety/geokrety-website#106]
Related to PR #77

Search by waypoint is now failing.

**Step to reproduce**
1. Open page https://boly38.gk.kumy.org/ruchy.php
1. Fill the "Waypoint" field with any known waypoint such as `GC5BRQK`

**Actual result**
* The lookup via https://boly38.gk.kumy.org/szukaj-ajax.php?skad=ajax&wpt=GC5BRQK report `{"tresc":"\"error\" No cache found"}`. A red cross is shown on the right.
* The `coordinates` field is filled with `undefined undefined`.

**Expected result**
* It should display some details information details about the cache on the right (green mark and country flag).
* The `coordinates` field should be filled with coordinates.

If there is no match at all, the red mark has to be displayed on the right, and a message indicating that "user has to fill coordinates manually" should be displayed.

**Comment by kumy** [comment]
@boly38 issue assigned to you.

Code is only deployed to rec

**Comment by boly38** [comment]
trying to google search "GC5BRQK" and seems to be related to the following cache:

https://www.geocaching.com/geocache/GC5BRQK : `Chemin de l'Escourachie`

trying to **ruchy.php search the following query: `Chemin de l`
- I'm able to see the  `Chemin de l'Escourachie` with code `OX5BRQK` as ruchy suggest on the right
- clicking on this code works fine.


I don't know what do you have in database (ex. prod) for `GC5BRQK` code.

I havent this code in my db. So I'm unable to reproduce the behavior you are talking about. 
According to prod, (cf. under) `GC5BRQK` code must be found.


In all cases, I need to fix the 'no result' case you provide: TODO remove `undefined undefined` (I am able to reproduce that issue)

---
I notice a strange behavior for  `GC5BRQK` (on PROD):

input `OX5BRQK`

![image](https://user-images.githubusercontent.com/3100576/41533046-a873ceb4-72f9-11e8-95fb-035acedfcfac.png)


input `GC5BRQK`

![image](https://user-images.githubusercontent.com/3100576/41533028-9bb2bfaa-72f9-11e8-8ca7-1a4194e173d3.png)

Why dont we have cache name, type, flag ?

@kumy could you provide me (insert  data) related to the following results :

`SELECT * FROM `gk-waypointy` where waypoint like '%BRQK'`

**Comment by boly38** [comment]
fix pushed onto [boly38](https://boly38.gk.kumy.org/ruchy.php) branch

**Comment by kumy** [comment]
@boly38 Here is the requested sql insert
`INSERT INTO `gk-waypointy` (`waypoint`, `lat`, `lon`, `alt`, `country`, `name`, `owner`, `typ`, `kraj`, `link`, `status`, `timestamp`) VALUES
('OX5BRQK', 43.69365, 6.86097, -32768, 'fr', 'Chemin de l\'Escourachie', 'kumy', 'Geocache|Traditional Cache', '', 'http://www.opencaching.com/#geocache/OX5BRQK', 1, '2014-08-29 04:56:02');`

But problem doesn't seem related to table `gk-waypointy`, IIRC, `GC` waypoints are looked up using table `gk-ruchy` :disappointed:

**Comment by kumy** [comment]
it happen here

https://github.com/geokrety/geokrety-website/blob/f0356ce836464d335fa7024e99ec319babfc2b98/website/szukaj-ajax.php#L75-L78

Then 

https://github.com/geokrety/geokrety-website/blob/67e54d0bb3666b5336a1b21a3d9548f323a0d3f9/website/waypoint_info.php#L38

**Comment by boly38** [comment]
waw part of legacy I forgot! good catch, will try to integrate it, thanks @kumy

**Comment by boly38** [comment]
I pushed a new version:
- we need to test with navicache (prefix 'N...')
- do we need to setup a cache name for the case we get back info from ruchy (we have only coordinates, alt, and builded cache link)

**Comment by kumy** [comment]
@boly38 please check your private mail for the necessary database extract.

Do you think it's stable enough to be pushed on prod, and threat navicache issue in another dedicated issue?

**Comment by kumy** [comment]
I have added a new commit to display the waypoint uppercase.

May you please squash all commits and open a PR? :)

**Comment by boly38** [comment]
I think you're ok to squash our commit together, will do that

**Comment by boly38** [comment]
#113 created

**Comment by boly38** [comment]
>(we're about ~25 commits away from master).

maybe could we add a label "deployed" to distinguish "closed issue" not deployed from closed issue deployed ? maybe to hard to maintain?

**Comment by kumy** [comment]
I m not opposed squash mine with yours, as long as there is one commit per feature/fix. Thanks!

On June 22, 2018 12:47:17 PM GMT+02:00, Boly38 <notifications@github.com> wrote:
>I think you're ok to squash our commit together, will do that
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/geokrety/geokrety-website/issues/106#issuecomment-399402752

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by kumy** [comment]
> maybe could we add a label "deployed" to distinguish "closed issue" not deployed from closed issue deployed ? maybe to hard to maintain?

Sure, we could give it a try

On June 22, 2018 1:07:58 PM GMT+02:00, Boly38 <notifications@github.com> wrote:
>>(we're about ~25 commits away from master).
>
>maybe could we add a label "deployed" to distinguish "closed issue" not
>deployed from closed issue deployed ? maybe to hard to maintain?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/geokrety/geokrety-website/issues/106#issuecomment-399407113

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
